Week2/alaa.js

```diff
@@ -0,0 +1,97 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
+//3
+var x;
+console.log('the value of my variable x will be: undefined');
+x = 7; // initialize the value of x
+console.log('the value of my variable x will be 7');
+console.log(x);
+//4
+var y;
+y = 'hyf';
+console.log('the value of y will be :hyf');
+console.log(y);
+var y = 'code';
+console.log('the value of y will be :code');
+console.log(y);
+//5
+var z = 7.25;
+console.log(z);
+var a = Math.floor(z);
+console.log(a);
+var max = Math.max(z, a);
+console.log(max);
```

Comment:
Again, please try to use `const` keyword always first followed by `let` and then very rarely should you use `var` keyword. Also, I would like you to find out the difference between `Math.floor` and `Math.round` 😉